src/components/display/Alert.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {Box, Snackbar} from '@material-ui/core';
+import {Alert as MuiAlert} from '@material-ui/lab';
+
+/**
+ * Alert component
+ */
+const Alert = ({id = 'alert', message = null}) => {
+  const [localMessage, setLocalMessage] = useState(message);
+
+  useEffect(() => {
+    if (localMessage !== message) {
+      setLocalMessage(message);
+    }
+  }, [message]);
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      setLocalMessage(null);
+    }
+  };
+
+  if (localMessage) {
+    return (
+      <Box id={id}>
+        <Snackbar
+          key={localMessage}
```

Comment:
⚠️ 

Key prop only required if only multiple elements in a list of React components.